tests/phpunit/CRM/Utils/HtmlToTextTest.php

```diff
@@ -56,8 +56,8 @@ public function htmlToTextExamples() {
   public function testHtmlToText($html, $text) {
     $output = CRM_Utils_String::htmlToText($html);
     $this->assertEquals(
-      trim($output),
       trim($text),
+      trim($output),
```

Comment:
Note that this is just correcting the order of the function parameters the expected output should be first then the actual recorded output